recipe/meta.yaml

```diff
@@ -0,0 +1,118 @@
+{% set name = "conda-store" %}
+{% set version = "2024.1.1" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  # This is only needed here in-repo; in conda-forge, sources from PyPI
+  - path: ../
+
+build:
+  number: 0
+
+outputs:
+  - name: conda-store
+    version: {{ version }}
+    build:
+      noarch: python
+      script: python -m pip install ./conda-store -vv --no-deps --no-build-isolation --no-index
+      entry_points:
+        - conda-store = conda_store.__main__:main
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+        - hatchling >=1.14.0
+        - hatch-vcs
+      run:
+        - __linux  # [linux]
+        - __osx    # [osx]
+        - __win    # [win]
+        - aiohttp
+        - click
+        - python >=3.8
+        - rich
+        - ruamel.yaml
+        - yarl
+      run_constrained:
+        - {{ pin_subpackage('conda-store-server', min_pin='x.x.x', max_pin='x.x.x') }}
+
+    test:
+      imports:
+        - conda_store
+      commands:
+        - conda-store --help
+
+  - name: conda-store-server
+    version: {{ version }}
+    build:
+      noarch: python
+      script: python -m pip install ./conda-store-server -vv --no-deps --no-build-isolation --no-index
+      entry_points:
+        - conda-store-server = conda_store_server.server.__main__:main
+        - conda-store-worker = conda_store_server.worker.__main__:main
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+        - hatchling >=1.14.0
+        - hatch-vcs
+      run:
+        - alembic
+        - celery
+        - conda
+        - conda-docker  # [linux]
+        - __linux  # [linux]
+        - __osx    # [osx]
+        - __win    # [win]
+        - conda-pack
+        - conda-lock
+        - fastapi
+        - filelock
+        - itsdangerous
+        - jinja2
+        - minio
+        - pydantic <2.0a0
+        - pyjwt
+        - python >=3.8
+        - python-docker
+        - python-multipart
+        - pyyaml
+        - redis-py
+        - requests
+        - sqlalchemy
```

Comment:
I see this is to [prevent accidental upgrades to v2](https://github.com/conda-incubator/conda-store/pull/480). Shouldn't we do it like this instead?

```suggestion
        - sqlalchemy <2.0a0
```

(In case there are security updates in the 1.x branch).

Reply:
Actually, there are: https://github.com/sqlalchemy/sqlalchemy/releases/tag/rel_1_4_49